### DISCUSSION TITLE
AI crew members should be given the ability to water/maintain plants upon being ordered to

### DISCUSSION THREAD

**Discussion by StruggleDuck** [FakeFishGames/Barotrauma#7123] [Ideas and suggestions]
I believe that in order to make gardening/planting more of a viable thing for smaller coop groups and **especially** singleplayer, players should be able to order crew members to water and maintain plants on board the sub. I know that personally, I find gardening pretty much useless as it is near impossible to maintain a plant for longer than one or two missions, as you have to juggle it with not only piloting the ship, but also with things like wreck salvages or ruin expeditions. I can also speak for pretty much everyone who I know/have met that are familiar with gardening/planting, about the frustrations of  having to be glued to the plants in order to make sure that they always have enough water, or to make sure that the they don't die due to fire or flooding (especially with pommegrenades). I do not believe that this would be a very difficult thing to add to the game, and would make an already existing feature in the game much more accessible for all players.

**Comment by MaaiG** [comment]
...And drag ammo from the dead ships! ...Which I pointed out to those of them who are next to me!
